src/com/projectkorra/projectkorra/firebending/lightning/Lightning.java

```diff
@@ -235,6 +242,9 @@ public void progress() {
 				final Location localLocation2 = new Location(this.player.getWorld(), d7, newY, d8);
 				playLightningbendingParticle(localLocation2);
 				this.particleRotation += 1.0D / d3;
+				if(ThreadLocalRandom.current().nextDouble() < .2){

```

Comment:
There should be a space between the if and opening parenthesis. Additionally, there should be a space between the closing parenthesis and the opening brace.

```java
if (ThreadLocalRandom.current().nextDouble() < .2) {
```